validator/__main__.py

```diff
@@ -46,21 +62,27 @@ def main():
                         type=str,
                         nargs='+',
                         help='Files to be validated')
+    parser.add_argument('-t',
+                        '--github-token',
+                        dest='token',
+                        type=str,
+                        help='Github access token to access private repo')
```

Comment:
I think I'd generally discourage passing a token on the command line, even in automation like this... never know where that will be recorded. Generally Jenkins makes it easy to provide credentials as env vars. Not sure it's a problem here, @thiagoalessio what do you think?